recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "plumbum" %}
-{% set version = "1.6.3" %}
+{% set version = "1.6.4" %}
 {% set sha256 = "0249e708459f1b05627a7ca8787622c234e4db495a532acbbd1f1f17f28c7320" %}
```

Comment:
You need to update the checksum too.

Reply:
I was hoping it was automatic in one of the bots. Will do.